src/interceptors/ClientRequest/utils/recordRawHeaders.ts

```diff
@@ -225,8 +188,8 @@ export function recordRawFetchHeaders() {
           inferredRawHeaders.push(...inferRawHeaders(args[1].headers))
         }
 
-        if (inferRawHeaders.length > 0) {
-          defineRawHeadersSymbol(request.headers, inferredRawHeaders)
+        if (inferredRawHeaders.length > 0) {
+          ensureRawHeadersSymbol(request.headers, inferredRawHeaders)
```

Comment:
Here I'm using `ensure` to prevent the raw headers symbol from being re-defined.